### ISSUE TITLE
Add darkmode

### ISSUE THREAD

**Issue by trentprynn** [trentprynn/habitapper.com#7]
It would be nice if the website had a dark mode and it used the user's preferred color scheme and perhaps had a slider to turn on / off like we're seeing on so many websites now

**Comment by trentprynn** [comment]
Hopefully bootstrap@6 adds support for it easily -- right now not worth the effort

reference: https://github.com/twbs/bootstrap/issues/27514

**Comment by hkusulja** [comment]
@trentprynn can you please provide more information about Bootstrap 6 ? is there somewhere beta that we can try / use ?
referenced issue is 3-4 years old, and does not provide information requested.

**Comment by trentprynn** [comment]
@hkusulja The current version of bootstrap is 5.1.3. Dark mode support is supposedly going to be officially supported in bootstrap version 6 which has no ETA

**Comment by hkusulja** [comment]
@trentprynn tx for the info about bootstrap v6., does it have some alpha/beta branch which we can maybe use or see ?

**Comment by trentprynn** [comment]
@hkusulja at this time, no

**Comment by TituxMetal** [comment]
Try [Tailwind Css](https://tailwindcss.com), it's really wonderful to use!
Tailwind Css also supports dark mode out of the box, no need to wait for a potential next version.

**Comment by trentprynn** [comment]
@TituxMetal agreed on tailwind being quite nice -- realistically I chose bootstrap as I use it all day at my job so I'm extremely proficient at getting what i need done. If I was going to start using tailwind I'd want to completely remove bootstrap from the project (don't want both as dependencies) which will obviously require redoing all styling. Not impossible as the website is pretty minimal but definitely non-trivial in terms of effort required
